helloword.py

```diff
@@ -1 +1,3 @@
-print("Hello Word!")
+import sys
+
+print("Hello {}!".format(sys.argv[1]))
```

Comment:
I do not really appreciate giving `sys.argv[1]` expression directly in the print statement... I would be better to create a variable from the `sys.argv[1]` expression